test/test/pending-handles.tap

```diff
@@ -0,0 +1,19 @@
+TAP version 13
+    # Subtest: ___/.*/~~~pending-handles.js child
+    ok 1 - this is ok
+    not ok 2 - received SIGTERM with pending event queue activity
+      ---
+      {"handles":{"0":{"msecs":100000,"type":"Timer"},"1":{"connectionKey":"6::::8000","events":{"0":"request","1":"connection","2":"clientError"},"type":"Server"},"2":{"events":{"0":"end","1":"finish","2":"_socketEnd","3":"free","4":"close","5":"agentRemove","6":"drain","7":"error","8":"data"},"type":"Socket"},"3":{"events":{"0":"end","1":"finish","2":"_socketEnd","3":"drain","4":"timeout","5":"error","6":"close","7":"data","8":"resume","9":"pause"},"type":"Socket"}},"requests":{"0":{"type":"FSReqWrap"}}}
```

Comment:
Oh, in case I misunderstood your question: I experimented with just dumping the request objects as-is to the YAML encoder, but you end up with a big glut of function strings and mostly-useless nonsense.  The goal of this output should be to give you a clue as to what might be going on, and if we just dump everything, there's a high likelihood of drowning out the useful bits in a bunch of noise.